### PR TITLE
Update to v30.0.0

### PR DESCRIPTION
**Pull request by marcoesters**
Needed for anaconda-linter

# Package information

Home: https://github.com/nexB/license-expression
Change log: https://github.com/nexB/license-expression/compare/v1.2...v30.0.0
Documentation: https://github.com/nexB/license-expression

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -15,11 +15,15 @@ build:
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
+  build:
+    - setuptools >=50
+    - setuptools_scm >=6
+    - wheel
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]: These packages should be in `host` not in `build`. For python packages, we use `host` when we are building them

File: recipe/meta.yaml [hunk 2]
```diff
@@ -15,11 +15,15 @@ build:
...
   host:
     - pip
     - python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:20]: Could you please add `toml` to `host`? See https://github.com/nexB/license-expression/blob/v30.0.0/pyproject.toml
`setuptools_scm[toml] >= 6` means both `setuptools_scm >= 6` **AND** `toml`

File: recipe/meta.yaml [hunk 3]
```diff
@@ -15,11 +15,15 @@ build:
...
   run:
-    - boolean.py >=3.6,<4.0.0
+    - boolean.py >=4.0
     - python
 
 test:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:29]: Could you please add:
```
  requires:
    - pip
  commands:
    - pip check
```

Comment (skupr-anaconda) [reply recipe/meta.yaml:29]: Missing `license_family: Apache`

File: recipe/meta.yaml [hunk 4]
```diff
@@ -15,11 +15,15 @@ build:
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:15]: Could you remove `noarch: python` and add `skip: True  # [py<36]`, see https://github.com/nexB/license-expression/blob/5f4febb6df2b644706839e42ba4310de9e2dccba/setup.cfg#L48

**Review by varlackc — COMMENTED** [review]
Since the package is using `pip` we should require `pip` on the test section and use `pip check`

File: recipe/meta.yaml (hunk 2 above)
Comment (marcoesters) [reply recipe/meta.yaml:20]: I was not aware of that - thank you!

**Comment by marcoesters** [comment]
I think I addressed all comments. Thank you @skupr-anaconda and @varlackc

**Review by skupr-anaconda — APPROVED** [review]

**Review by varlackc — APPROVED** [review]
LGTM

**Pull request merged**